ecs-agent/tmds/handlers/fault/v1/handlers/handlers.go

```diff
@@ -42,12 +42,15 @@ import (
 )
 
 const (
+	// Request types
 	startFaultRequestType       = "start %s"
 	stopFaultRequestType        = "stop %s"
 	checkStatusFaultRequestType = "check status %s"
-	invalidNetworkModeError     = "%s mode is not supported. Please use either host or awsvpc mode."
-	faultInjectionEnabledError  = "enableFaultInjection is not enabled for task: %s"
-	requestTimedOutError        = "%s: request timed out"
+	// Error messages
+	internalError              = "internal error"
```

Comment:
Might be useful to also include the request type in the error message that we respond back to. thoughts? @tshan2001 @xxx0624 @amogh09

Reply:
When the customer curls the endpoint, they would need to specify which fault API they're calling. So they would see something like this: 
```
% curl -X POST ${ECS_AGENT_URI}/fault/v1/network-latency/stop
{"Error":"internal error"}
```

If we add the fault type, it would look like this:
```
% curl -X POST ${ECS_AGENT_URI}/fault/v1/network-latency/stop
{"Error":"network latency: internal error"}
```

Since the type is already specified in the API URL, would it be necessary to include that in the error response?

Reply:
That's true, I was thinking in the case where there's multiple request coming in and out and how to better track it. I don't too strong of an opinion on this actually